cvat/schema.yml

```diff
@@ -7140,6 +7866,66 @@ components:
             type: string
             minLength: 1
           writeOnly: true
+    PatchedQualitySettingsRequest:
+      type: object
+      properties:
+        iou_threshold:
+          type: number
+          format: double
+          description: Used for distinction between matched / unmatched shapes
+        oks_sigma:
+          type: number
+          format: double
+          description: |
+            Like IoU threshold, but for points.
+            The percent of the bbox area, used as the radius of the circle around the GT point,
+            where the checked point is expected to be.
+            Read more: https://cocodataset.org/#keypoints-eval
+        line_thickness:
+          type: number
+          format: double
+          description: |
+            Thickness of polylines, relatively to the (image area) ^ 0.5.
+            The distance to the boundary around the GT line,
+            inside of which the checked line points should be
+        low_overlap_threshold:
+          type: number
+          format: double
+          description: Used for distinction between strong / weak (low_overlap) matches
+        oriented_lines:
+          type: boolean
+          description: Indicates that polylines have direction
+        line_orientation_threshold:
+          type: number
+          format: double
+          description: |
+            The minimal gain in the GT IoU between the given and reversed line directions
+            to consider the line inverted. Only useful with the 'oriented_lines' parameter
```

Comment:
The associated boolean parameter is described differently for each of `line_orientation_threshold`, `group_match_threshold` and `object_visibility_threshold`. Please use the same format everywhere.

Reply:
The parameters are different, I don't see how the same format can be used without hurting understanding.

Reply:
Ah, if your suggestion was about this part, then ok.

Reply:
Updated